.github/workflows/haskell.yml

```diff
@@ -21,6 +21,22 @@ jobs:
         ghc-version: ${{ matrix.ghc }}
```

Comment:
Are you seeing a check warning on the first line? Any idea what it indicates?

Reply:
I do in early check runs, but not on the later ones?

Reply:
Like so?

<img width="667" alt="Screen Shot 2019-11-04 at 11 00 29 AM" src="https://user-images.githubusercontent.com/59671/68135894-5601a580-fef2-11e9-84ac-30cb9f745ac3.png">

Hopefully it’s benign…